<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MstClass</class>
 <widget class="QMainWindow" name="MstClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mst</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>321</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>输入</string>
    </property>
    <widget class="QTextEdit" name="textEdit_input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_import">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>101</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>101</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;格式:限数字&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;a b v&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;每行一条边&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rand">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>109</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>样例</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>711</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>输出</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>490</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>原图</string>
     </property>
     <widget class="QLabel" name="label_graph">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>201</width>
        <height>431</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>预览</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>411</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>结果</string>
     </property>
     <widget class="QTextEdit" name="textEdit_result">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>390</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_result">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>191</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string>输出</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_sum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mst:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_kruskal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Kruskal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_prim">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prim</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>计算</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>btn_kruskal</zorder>
   <zorder>btn_prim</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mst.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_import</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnSaveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>-30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnLoadClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnOkClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>-35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_rand</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnRandClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>-46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_prim</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnPrimClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_kruskal</sender>
   <signal>clicked()</signal>
   <receiver>MstClass</receiver>
   <slot>btnKrusalClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnShowClick()</slot>
  <slot>btnSaveClick()</slot>
  <slot>btnLoadClick()</slot>
  <slot>btnOkClick()</slot>
  <slot>btnRandClick()</slot>
  <slot>btnPrimClick()</slot>
  <slot>btnKrusalClick()</slot>
 </slots>
</ui>
